### ISSUE TITLE
BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding

### ISSUE THREAD

**Issue by jinmang2** [jiphyeonjeon/season3#3]
## 집현전 중급반 스터디
- 2022년 3월 20일 일요일 9시
- 박범수님 임승원님 최성욱님 이경로님 발표
- 논문 링크: https://arxiv.org/abs/1810.04805

> ### Abstract
> We introduce a new language representation model called BERT, which stands for Bidirectional Encoder Representations from Transformers. Unlike recent language representation models, BERT is designed to pre-train deep bidirectional representations from unlabeled text by jointly conditioning on both left and right context in all layers. As a result, the pre-trained BERT model can be fine-tuned with just one additional output layer to create state-of-the-art models for a wide range of tasks, such as question answering and language inference, without substantial task-specific architecture modifications.
BERT is conceptually simple and empirically powerful. It obtains new state-of-the-art results on eleven natural language processing tasks, including pushing the GLUE score to 80.5% (7.7% point absolute improvement), MultiNLI accuracy to 86.7% (4.6% absolute improvement), SQuAD v1.1 question answering Test F1 to 93.2 (1.5 point absolute improvement) and SQuAD v2.0 Test F1 to 83.1 (5.1 point absolute improvement).

**Comment by jinmang2** [comment]
- 발표 링크: https://www.youtube.com/watch?v=DBdikhzBJiY&list=PL2tRglRS_GqjCcBVM26gYNdoOnX12lLIh&index=2&t=4s
